packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -181,23 +177,30 @@ $logo-borderColor                         : $color-border-default;
   }
 }
 
+.c-restaurantCard:hover {
+    .c-restaurantCard-name {
+        text-decoration: underline;
+    }
+}
```

Comment:
```suggestion
.c-restaurantCard {
...

  &:hover {
      .c-restaurantCard-name {
          text-decoration: underline;
      }
  }
}
```

we need this nesting anyway so might be worth moving it to the `.c-restaurantCard` block

Reply:
ah, so are the nesting rules about writing out class names not applied to states like hover?

Reply:
no, because we still can easily search for the class name in the IDE. No need to repeat the class name in this case.

Reply:
So there're basically 2 rules regarding this:
- avoid nesting when possible. it's not possible here
- write the full class name like `.c-restaurantCard-name` not `&-name`